src/test/java/model/User.java

```diff
@@ -0,0 +1,52 @@
+package model;
+
+import java.util.Objects;
+
+public class User {
```

Comment:
please use Lambook (https://projectlombok.org) annotations instead of methods. Please look into @Data & @AllArgsConstructor